api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -5,264 +5,160 @@
-import io.opentelemetry.context.propagation.DefaultContextPropagators;
-import io.opentelemetry.internal.Obfuscated;
-import io.opentelemetry.metrics.DefaultMeterProvider;
 import io.opentelemetry.metrics.Meter;
 import io.opentelemetry.metrics.MeterProvider;
-import io.opentelemetry.metrics.spi.MeterProviderFactory;
-import io.opentelemetry.trace.DefaultTracerProvider;
 import io.opentelemetry.trace.Tracer;
 import io.opentelemetry.trace.TracerProvider;
-import io.opentelemetry.trace.propagation.HttpTraceContext;
-import io.opentelemetry.trace.spi.TracerProviderFactory;
-import java.util.Objects;
-import java.util.ServiceLoader;
-import javax.annotation.Nullable;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * This class provides a static global accessor for telemetry objects {@link Tracer}, {@link Meter}
- * and {@link BaggageManager}.
+ * The entrypoint to telemetry functionality for tracing, metrics and baggage.
+ *
+ * <p>A global singleton can be retrieved by {@link #get()}. The default for the returned {@link
+ * OpenTelemetry}, if none has been set via {@link #set(OpenTelemetry)} will be created with any
+ * {@link io.opentelemetry.spi.OpenTelemetryFactory}, {@link
+ * io.opentelemetry.trace.spi.TracerProviderFactory}, {@link
+ * io.opentelemetry.metrics.spi.MeterProviderFactory}, or {@link
+ * io.opentelemetry.baggage.spi.BaggageManagerFactory} found on the classpath, or otherwise will be
+ * default, with no-op behavior.
  *
- * <p>The telemetry objects are lazy-loaded singletons resolved via {@link ServiceLoader} mechanism.
+ * <p>If using the OpenTelemetry SDK, you may want to instantiate the {@link OpenTelemetry} to
+ * provide configuration, for example of {@code Resource} or {@code Sampler}. See {@code
+ * OpenTelemetrySdk} and {@code OpenTelemetrySdk.newBuilder} for information on how to construct the
+ * SDK {@link OpenTelemetry}.
  *
  * @see TracerProvider
- * @see MeterProviderFactory
- * @see BaggageManagerFactory
+ * @see MeterProvider
+ * @see BaggageManager
  */
 @ThreadSafe
-public final class OpenTelemetry {
-  private static final Object mutex = new Object();
+public interface OpenTelemetry {
 
-  @Nullable private static volatile OpenTelemetry instance;
+  /**
+   * Returns the registered global {@link OpenTelemetry}. If no call to {@link #set(OpenTelemetry)}
+   * has been made so far, a default {@link OpenTelemetry} composed of functionality any {@link
+   * io.opentelemetry.spi.OpenTelemetryFactory}, {@link
+   * io.opentelemetry.trace.spi.TracerProviderFactory}, {@link
+   * io.opentelemetry.metrics.spi.MeterProviderFactory}, or {@link
+   * io.opentelemetry.baggage.spi.BaggageManagerFactory} found on the classpath, or otherwise * will
+   * be default, with no-op behavior.
+   *
+   * @throws IllegalStateException if a provider has been specified by system property using the
+   *     interface FQCN but the specified provider cannot be found.
+   */
+  static OpenTelemetry get() {
```

Comment:
Although this is nice, I don't _really_ want to be writing this every time I use `OpenTelemetry` ;)

```java
OpenTelemetry.get().getTracerProvider().etc()
OpenTelemetry.get().getMeterProvider().etc()
```

I'd rather provide this `get()` method **plus** shortcuts (I know we are not fan of them, but hey, they are in the same class):

```java
OpenTelemetry.getTracerProvider().etc()
OpenTelemetry.getMeterProvider().etc()
```